apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -389,7 +389,8 @@ handle_cast(do_prepare_replica_2_9, State) ->
 				%% storage modules are commonly set up with.
```

Comment:
Suggesting a change: `since the 2.9 partition is slightly bigger than the recall partitition...` -> `since the storage module may be configured to be smaller than the partition.`

Reply:
Updated! https://github.com/ArweaveTeam/arweave/pull/680/commits/6811ceae2fa0632e5fe830b34ebb1fd85585af55